### PR TITLE
Add KLDiv loss

### LINKED ISSUE #870: KLDivLoss support

### PR DESCRIPTION
**Pull request by pglusacTT**
Add support for KL divergence loss.

Closes #870

### CODE REVIEW

File: forge/forge/op/loss.py [hunk 1]
```diff
@@ -119,3 +119,18 @@ def forward(self, predictions, labels):
         square = Multiply("square", diff, diff)
         loss = reduce_loss(self.reduction, square)
         return loss
+
+
+class KLDivLoss(ForgeModule):
```
Comment (pmarkovicTT) [inline forge/forge/op/loss.py:124]: nit: Add description here as well following convention for other losses

**Review by pmarkovicTT — APPROVED** [review]
nit comment, but lgtm

**Review by pilkicTT — APPROVED** [review]
It would be good to use `verify()` for checking the accuracy of the results. Also, for the rest of the loss tests (wherever possible). You can do it in a separate PR...

```python
verify(inputs, torch_loss, forge_loss, VerifyConfig(value_checker=AllCloseValueChecker(rtol=1e-5)))
```
